src/web/lib/messagePicker.ts

```diff
@@ -0,0 +1,103 @@
+import { record } from '@root/src/web/browser/ophan/ophan';
+
+export type MaybeFC = React.FC | null;
+type ShowMessage = (meta?: any) => MaybeFC;
+
+export type CanShowResult = {
+	result: boolean;
+	meta?: any;
+};
+
+export type Message = {
```

Comment:
Is this really a `Message`, or is this more like one of the following:
* channel
* message handler
* or even more slot configuration?

Reply:
Yeah I see what you mean, `Message` isn't quite right. It represents the possibility of showing a message, in this slot.